content/4_Gold/Combinatorics.problems.json

```diff
@@ -137,8 +137,8 @@
       "isStarred": false,
       "tags": ["Combinatorics"],
       "solutionMetadata": {
-        "kind": "USACO",
-        "usacoId": "1283"
+        "kind": "internal",
+        "hasHints": true
```

Comment:
hm? when did we add this field 👀

Reply:
a while back lol in like #3740 but no one used it :(

Reply:
haha i'm so sorry daniel